fameal/ape-tax
# compatibility: Replace ethers import

## What it does ✨
It's a minimal pull request to fix some issue while trying to run the project locally.
- Ethers could be imported as default import in [ethers V4](https://docs.ethers.io/v4/getting-started.html), but should be imported this way in [ethers V5](https://docs.ethers.io/v5/getting-started/) : `import {ethers} from "ethers";`
- The `BigNumber` utils from ethers V5 does not require a `new` keyword

## How to test ✅
The project should be able to start with a `yarn && yarn run serve`, and ethers should be correctly loaded.

## Resources ✏️
- [Ethers](https://docs.ethers.io/v5/getting-started/)
- [Big Numbers](https://docs.ethers.io/v5/api/utils/bignumber/)
